subsys/bluetooth/host/gatt.c

```diff
@@ -693,11 +697,93 @@ static ssize_t cf_write(struct bt_conn *conn, const struct bt_gatt_attr *attr,
 	return len;
 }
 
+#if defined(CONFIG_BT_USE_PSA_API)
```

Comment:
I think @jori-nordic was saying that the cryptographic specifics should be abstracted away from `gatt.c`.

Reply:
I'm not sure. He mentioned a `utility function` in this case, whereas in previous comments he explicitly said to create a separate file. As you (@jori-nordic) prefer, I can fix, this just let me know :)

Reply:
Sure, I can work on that!
Just out of my curiosity would this new PR still be in scope of the next release or will it be postponed after the code freeze?

Reply:
you should post it right away but it will be merged after, yes.